Source/ACE/Managers/WorldManager.cs

```diff
@@ -10,14 +10,17 @@
 using ACE.Entity;
 using ACE.Network;
 
+using log4net;
+
 namespace ACE.Managers
 {
     public static class WorldManager
     {
-        // commented: unused.  uncomment if you'll use it.
-        // private static uint sessionTimeout = 150u; // max time between packets before the client disconnects
+        private static readonly ILog log = LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
 
-        private static readonly List<Session> sessionStore = new List<Session>();
+        public const ushort ServerId = 0xB;

```

Comment:
Why did you chose 0xB?  Mostly curious, but this feels like a config thing to me for people that want to host multiple worlds.